app/services/provider_email_service.rb

```diff
@@ -0,0 +1,41 @@
+class ProviderEmailService
+  include Rails.application.routes.url_helpers
+
+  def initialize(application)
+    @application = application
+  end
+
+  def send_email
+    # Must use bang version `deliver_later!` or failures won't be retried by sidekiq
+    CitizenCompletedMeansMailer.notify_provider(*mailer_args).deliver_later!
+  end
+
+  private
+
+  attr_reader :application
+
+  def mailer_args
+    [
+      application,
+      provider.name,
+      applicant.full_name,
+      application_url
+    ] + with_provider_email
+  end
+
+  def with_provider_email
+    HostEnv.staging? ? [] : [provider.email]
+  end
```

Comment:
Can we turn this round so that it passes in the provider email for host env production, and not for everything else?  I feel that's a bit safer.